_maps/travis_map_testing.dm

```diff
@@ -1,156 +0,0 @@
-// This is for Travis testing. DO NOT SET THIS AS THE GAME'S MAP NORMALLY!
```

Comment:
Is there anything to take this thing's place in terms of functionality?

Reply:
There is a python script inside `tools/travis` which will automatically generate this file based on the DMM files that exist, removing the need for updating the file, as well as making sure that it never becomes stale.

Reply:
Ah, got it, thanks!